app/javascript/src/components/Dashboard/Notes/Card.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "neetoicons";
+import { Typography, Tag, Dropdown, Avatar, Tooltip } from "neetoui";
+
+import { calculateCreatedAgo, calculateDateToWeekday } from "./utils";
+
+const Card = ({ note }) => {
+  const { MenuItem } = Dropdown;
+  const { title, description, created_at } = note;
+  return (
+    <div className="neeto-ui-shadow-xs neeto-ui-rounded-none neeto-ui-border-gray-300 mb-4 w-full border py-3 px-4">
+      <div className="flex items-center justify-between">
+        <Typography className="mb-px leading-6" style="h4">
+          {title}
+        </Typography>
+        <Dropdown buttonSize="small" buttonStyle="text" icon={MenuVertical}>
+          <MenuItem.Button>Edit</MenuItem.Button>
+          <MenuItem.Button>Delete</MenuItem.Button>
+        </Dropdown>
```

Comment:
What's the advantage of keeping each of the items in a Dropdown as child of `MenuItem.Button`? Please create a video and show me the UI changes and why you preferred to keep this over something like `li` tag.

Reply:
Actually, I referred to the neetoUI docs when writing this code. I didn't know I could have used li in place of it.

Here is the link to the video.
https://www.loom.com/share/966ef24f615642ecb885908e3d7fe5dd

Reply:
@ravijyoti3 Raise an issue in https://github.com/bigbinary/neeto-ui.

The issue should be more of like a question regarding why we have used `MenuItem.Button` over a simple `li`. Point out that when you tried both from UI, there weren't any significant changes which were seen.